wpeverest/user-registration
# Tweak/toggle field options

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a toggle feature in the `Field Options` section. For example, `General Settings` can be toggled by clicking on the header or the icon on the far right.



### How to test the changes in this Pull Request:

1. Create a form
2. Add a field
3. Check if the toggle feature is working in the following sections:

`Fields` Tab:
- `Default User Fields` section
- `Extra Fields` section

`Field Options` Tab:
- `General Settings` section
- `Advance Settings` section

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Toggle feature in the `Field Options` tab
